spec/features/posting_peeps_spec.rb

```diff
@@ -0,0 +1,43 @@
+feature 'Posting peeps' do
+
+  let!(:user) do
+    User.create(name: 'Bo Peep',
+                username: 'BoPeep789',
+                email: 'bo@peep.com',
+                password: 'sheepies',
+                password_confirmation: 'sheepies')
+    log_in(email: 'bo@peep.com', password: 'sheepies')
```

Comment:
this line should be in a `before` block.  The `let` statement is intended to define a test object (i.e. `user`).